### PR TITLE
Sell coinify

### PR DESCRIPTION
**Pull request by prwelber**

### CODE REVIEW

File: assets/css/utilities/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.type-xs { font-size: .75em; }
```
Comment (plondon) [inline assets/css/utilities/_typography.scss]: Please remove this file. If you have a change _typography.scss please make a PR to https://github.com/blockchain/brand-assets/blob/css/sass/utilities/_typography.scss

File: assets/css/utilities/_colors.scss [hunk 2]
```diff
@@ -58,6 +58,9 @@ $security-green: $jade;
 $security-yellow: $dandelion;
 $bright-green:  $jade !important; //override Security Center nested css selectors
 
+//opacity
+.opacity-5 { opacity: .5; }
```
Comment (plondon) [inline assets/css/utilities/_colors.scss]: Added to blockchain-css https://github.com/blockchain/brand-assets/blob/css/sass/utilities/_colors.scss#L16-L20. Please remove this.

File: app/partials/buy-sell.pug [hunk 3]
```diff
@@ -36,26 +36,27 @@
...
-        //-                 sell="sell(null, {quote: quote, btc: btc, fiat: fiat, currency: transaction.currency})")
-        //- .flex-column.width-50.mlvl.mt-55
-        //-   kyc-status(state="kyc.state" on-trigger="openKyc()" limits="limits")
+        sell-quick-start(limits="limits",
+                        transaction="transaction",
+                        disabled="status.disabled",
+                        currency-symbol="currencySymbol",
+                        modal-open="status.modalOpen",
+                        change-currency="changeCurrency(currency)"
+                        trading-disabled="getIsTradingDisabled()"
```
Comment (plondon) [inline app/partials/buy-sell.pug]: Is sell's disabled logic the same as buy's?

File: app/templates/trade.pug [hunk 4]
```diff
@@ -28,10 +33,12 @@ tr.border
...
           i.ti-search(ng-click="trade.expireQuote()") QA: Now
         br
-    span.display-inline-block(ng-class="{italic: pending && expiredQuote, 'tilda-before': pending && expiredQuote}") {{ pending && expiredQuote ? btcExpected / 100000000 : (trade.outAmount || trade.outAmountExpected) / 100000000 }} {{ trade.outCurrency }}
+    span.display-inline-block(ng-show="trade.medium === 'blockchain'") {{ trade.inAmount / 100000000 }} {{ trade.inCurrency }}
+    span.display-inline-block(ng-hide="trade.medium === 'blockchain'" ng-class="{italic: pending && expiredQuote, 'tilda-before': pending && expiredQuote}") {{ pending && expiredQuote ? btcExpected / 100000000 : (trade.outAmount || trade.outAmountExpected) / 100000000 }} {{ trade.outCurrency }}
   td.type-lbl
     span(ng-hide="status.gettingQuote")
-      span.em-600 {{ trade.sendAmount / 100 | format }} {{ trade.inCurrency }}
+      span.em-600(ng-hide="trade.medium === 'blockchain'") {{ trade.sendAmount / 100 | format }} {{ trade.inCurrency }}
+      span.em-600(ng-show="trade.medium === 'blockchain'") {{ trade.outAmountExpected / 100 }} {{ trade.outCurrency }}
```
Comment (jtormey) [inline app/templates/trade.pug]: We have some filters that handle currency conversion, could you use those here instead?

File: assets/js/controllers/coinify/CoinifyBankLink.controller.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  .module('walletApp')
+  .controller('CoinifyBankLinkController', CoinifyBankLinkController);
+
+function CoinifyBankLinkController ($scope, Alerts, buySell, $q) {
+  $scope.transaction = $scope.$parent.transaction;
+  $scope.bankAccounts = $scope.$parent.bankAccounts.filter(b => b.account.currency === $scope.transaction.currency); // only show banks with selected currency
+
+  $scope.selecting = true;
+
+  $scope.$parent.$parent.selectedBankAccount = $scope.selectedBankAccount;
```
Comment (jtormey) [inline assets/js/controllers/coinify/CoinifyBankLink.controller.js]: Would rather pass state through component props than by accessing $parent.$parent

File: assets/js/services/country.service.js [hunk 6]
```diff
@@ -229,8 +229,44 @@ function country () {
                         {'Name': 'Western Sahara', 'Code': 'EH'},
                         {'Name': 'Zambia', 'Code': 'ZM'}];
 
+  const sepaCountryCodes = [{name: 'Austria', code: 'AT'},
```
Comment (jtormey) [inline assets/js/services/country.service.js]: Should we put these in wallet options file?

**Pull request merged**